src/Kestrel.Core/Internal/HttpProtocols.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Core
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal
 {
     [Flags]
     public enum HttpProtocols
```

Comment:
This one can go full internal until 2.2. No need to change the namespace.

Reply:
The only problem with that is that the UseHttpServer IConnectionBuilder methods will need to also change from pubternal to actually internal. @davidfowl Are you OK with that?

Reply:
For reference, these are all the additional changes that would need to be made to make this internal: https://github.com/aspnet/KestrelHttpServer/commit/27c8ea4e562668ca22ba6c7f69ee47d3df22f10d

Reply:
Though most of those namespace changes are reverts.

Reply:
True. Focus on the methods/ctor/and property that are now also internal.

Reply:
Ugh, I wanted to demo this thing but thats ok.

Reply:
Or we could just keep it pubternal like it is in this PR.